lxd/db/instances.go

```diff
@@ -1070,7 +1070,7 @@ SELECT instances_snapshots.name
   JOIN instances ON instances.id = instances_snapshots.instance_id
   JOIN projects ON projects.id = instances.project_id
 WHERE projects.name=? AND instances.name=?
-ORDER BY date(instances_snapshots.creation_date)
+ORDER BY datetime(instances_snapshots.creation_date)
```

Comment:
Do we even need to wrap the order by column with a function at all?

Reply:
Should be `ORDER BY instances_snapshots.creation_date, instances_snapshots.id`

Thanks!

Reply:
Yeah, I looked for anywhere else where we sort by timestamp and it's the only spot where we use `date()`.
I suspect sorting by `creation_date` directly should have sqlite sort by Unix epoch which should then order properly too and with less processing needed.

Reply:
When I was doing testing earlier, directly sorting on the `creation_date` did in fact get the same result, just since there was already. So is the desired thing to not cast to `datetime` at all?

Reply:
Yes and then sort by id in cases where both times are the same.

Reply:
I'll put up a PR for this change.

Reply:
I noticed also that your change does not fix `lxc info <instance>` snapshot order output anyway, but I'll fix that as well.